### ISSUE TITLE
Steam shortcuts fail due to bad exe path

### ISSUE THREAD

**Issue by MattyXarope** [ebenbruyns/junkstore#7]
Some Steam shortcuts created by the plugin fail upon opening.

I've realized that this is a game exe path issue.

As of now, the scripts create a wrong path to the executable file, seen here in Target:
![Screenshot_20231205_063518](https://github.com/ebenbruyns/junkstore/assets/30245871/3f94c528-2079-4700-aa1b-eed4a6dbc5db)

After updating the Target path to the correct exe location for the game as seen here:

![Screenshot_20231205_063539](https://github.com/ebenbruyns/junkstore/assets/30245871/35e7bffb-5f46-459c-8fe9-4bb353804a09)

The game boots. 

However, it seems some games work fine with the current paths, such as Among Us:

![Screenshot_20231205_074914](https://github.com/ebenbruyns/junkstore/assets/30245871/73eb0ab5-766d-4a33-a917-da12411249aa)

Here the game boots.

So I think the best thing that would work with all games would be to stick with setting the full path which is the Start In + Target paths combined.

**Comment by MattyXarope** [comment]
![Screenshot_20231205_063518](https://github.com/ebenbruyns/junkstore/assets/30245871/3f94c528-2079-4700-aa1b-eed4a6dbc5db)
![Screenshot_20231205_063539](https://github.com/ebenbruyns/junkstore/assets/30245871/35e7bffb-5f46-459c-8fe9-4bb353804a09)

**Comment by ebenbruyns** [comment]
This all comes from legendary, so it will take a bit of deeper investigation. I'll get on this as soon as I can

**Comment by ebenbruyns** [comment]
I just realised I have this in my library, I'll get it installed and see what I can do to fix this, the solution you proposed might work.

**Comment by ebenbruyns** [comment]
I've made a fix, it's not in testing yet, but the scripts should work if you replace them for now. You will have to reinstall prey to get the shortcut created again. I've changed the plugin code to update automatically when you run it from the plugin - so it's self healing, or continually breaking depending on how you look at it...

I need to figure out a better build pipeline and deployment for testing, I can't go ask the Decky guys to approve every change just for testing. Let me know how you go with this.
